code/scripting/api/libs/graphics.cpp

```diff
@@ -493,6 +493,16 @@ ADE_FUNC(setColor, l_Graphics, "number Red, number Green, number Blue, [number A
 	return ADE_RETURN_NIL;
 }
 
+ADE_FUNC(getColor, l_Graphics, nullptr, "Gets the active 2D drawing color", "number, number, number, number" , "rgba color which is currently in use for 2D drawing")
+{
+	
```

Comment:
I'd get rid of this newline at the top of the function but otherwise this looks good.

(I'm assuming you tested the functionality here; I haven't tried returning values from functions in this way.)

Reply:
Yep, I tested the functionality by reproducing the use case I described in the PR. 

Just pushed a commit removing the newline.